### PR TITLE
Do not use AssetFetcher on iOS

### PR DESCRIPTION
**Pull request by jisqyv**
Do not use AssetFetcher on iOS, It isn't implemented there yet.

Change-Id: I98a281249576e25893422bb957f2659182ee6f59

### CODE REVIEW

**Comment by jisqyv** [comment]
@ewpatton I'm going to put this change up on ai2-test (with the rest of the work intended for the release). I'll let you know when it is there so you can test on a real iOS device. I had to fudge it...

**Comment by jisqyv** [comment]
hmmm. I put this on ucr, but it really should be on master... I'll move it.

**Comment by jisqyv** [comment]
This code is now live on ai2-test (ai2-test-20220603)

File: appinventor/blocklyeditor/src/replmgr.js [hunk 1]
```diff
@@ -1462,6 +1462,7 @@ Blockly.ReplMgr.getFromRendezvous = function() {
                 rs.versionurl = 'http://' + json.ipaddr + ':8001/_getversion';
                 rs.baseurl = 'http://' + json.ipaddr + ':8001/';
                 rs.android = (json.os || 'Android').toLowerCase() !== 'ios';
+                rs.hasfetchassets = rs.android
```
Comment (ewpatton) [inline appinventor/blocklyeditor/src/replmgr.js:1465]: Not that it matters technically, but there is a missing semicolon at the end of the line here.

**Review by ewpatton — APPROVED** [review]
One minor syntax thing that's not really an issue, otherwise LGTM.

**Comment by ewpatton** [comment]
Tested on an iPhone X running iOS companion 2.60 build 8.

File: appinventor/blocklyeditor/src/replmgr.js (hunk 1 above)
Comment (jisqyv) [reply appinventor/blocklyeditor/src/replmgr.js]: Fixed, good catch, thanks.

**Pull request merged**